cmd/wallet-web/src/pages/LoginHandle.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div>
+    <p>Redirecting...</p>
+  </div>
+</template>
+
+<script>
+import { mapActions, mapGetters } from 'vuex';
+
+export default {
+  name: 'LoginRedirect',
+  props: {
+    provider: {
+      type: String,
+      default: null,
+    },
+  },
+  data() {
+    return {
+      userinfo: null,
+    };
+  },
+  created: async function () {
+    await this.checkIfUserLoggedIn();
+    if (this.userInfo.ok) {
+      this.updateUserOnboard();
+      window.top.close();
+    } else {
+      window.location.href = this.serverURL() + '/oidc/login?provider=' + this.provider;
+    }
+  },
+  methods: {
+    ...mapActions({
+      updateUserOnboard: 'updateUserOnboard',
+    }),
+    ...mapGetters(['serverURL']),
+    checkIfUserLoggedIn: async function () {
+      console.log('checking userinfo');
+      this.userInfo = await fetch(this.serverURL() + '/oidc/userinfo', {
+        method: 'GET',
+        credentials: 'include',
+      });
+    },
+  },
+};
+</script>
+
+<style scoped></style>
```

Comment:
Is this needed?

Reply:
vue file by default create this, i have removed